notificationWorker/worker/worker.go

```diff
@@ -94,10 +98,41 @@ func (w *Worker) notifyInvoiceParties(ctx goContext.Context, invoice entity.Invo
 			return err
 		}
 
-		err = w.mail.SendInvoice(ctx, invoice.Id, emails, invoiceXml)
+		invoicePdfFile := visualization.Generate(&invoice)
+		var b bytes.Buffer
+		err = invoicePdfFile.Write(&b)
 		if err != nil {
 			return err
 		}
+		invoicePdf := b.Bytes()
+
+		for _, uri := range uris {
+			msg, err := upvs.CreateMsg(w.config.MailSenderUri,
+				uri,
+				fmt.Sprintf("Invoice %d", invoice.Id),
+				fmt.Sprintf("Invoice %d published.\nhttps://dev.einvoice.mfsr.sk/invoices/%d", invoice.Id, invoice.Id),
+				fmt.Sprintf("invoice-%d", invoice.Id),
+				invoiceXml,
+				invoicePdf,
+			)
+			if err != nil {
+				context.GetLogger(ctx).
+					WithField("error", err.Error()).
+					Error("worker.checkInvoices.notifyInvoiceParties.createMessage.failed")
+			}
```

Comment:
in case of error we should not proceed with sending message
We should even return error, so we don't mark in DB this invoice is fully notified.
Later we can implement some complicated mechanism to avoiding sending some messages twice (but still better than no notification)